Model/app-model.js

```diff
@@ -1,7 +1,19 @@
 const db = require('../db/connection')
+const { articleData } = require('../db/data/test-data')
+
 
```

Comment:
Couple of things

1.  Start to extract functionality based on different types of endpoints into separate model and controller files. Rather than having one controller file and one model file for your whole app you should have different model and controller files relating to request types. For example one for requests relating to topics and another for those relating to articles. This will make your application much easier to debug

2. How come you are bringing in your test article data into the model? This data should only be required into the test environment, which is your test file.

Reply:
initially i forgot about testing for results.rows.length === 0 and instead was testing if article_id was greater than articleData.length and then forgot to delete articleData after i changed it